### PR TITLE
[#10153] result page collapsable box ui issues

### LINKED ISSUE #10153: Result Page: Collapsable box UI issues
**In general**

The clickable parts of the boxes should change the cursor to a pointer when hovered over. This is a common issue at many places, e.g. `instructor-student-list-page`, `instructor-student-records-page`, and `instructor-session-result-page` as well.

**Instructor session result page**

1. When grouped by questions, clicking on the text does not collapse the box. However, clicking on the text collapses the box when grouped by any other way.
output

2. When grouped by questions at an MCQ question, click on "more" and try to expand the box. Only certain parts of the box is clickable.
output

### PR DESCRIPTION
**Pull request by Adrian27045**
Fixes #10153

1.

2. 

The clickable parts of the boxes change the cursor to a pointer
When grouped by questions, clicking on the text collapses the box.

### CODE REVIEW

File: src/web/app/pages-instructor/instructor-session-result-page/instructor-session-result-question-view.component.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="card margin-bottom-15px" *ngFor="let question of questionsOrder">
   <div class="card-header bg-info text-white course-tab-header" (click)="toggleAndLoadTab.emit(question.question.feedbackQuestionId)">
-    <tm-question-text-with-info [questionNumber]="question.question.questionNumber" [questionDetails]="question.question.questionDetails" (click)="$event.stopPropagation()"></tm-question-text-with-info>
+    <tm-question-text-with-info [questionNumber]="question.question.questionNumber" [questionDetails]="question.question.questionDetails" (click)="toggleAndLoadTab.emit()"></tm-question-text-with-info>
```
Comment (ccyccyccy) [inline src/web/app/pages-instructor/instructor-session-result-page/instructor-session-result-question-view.component.html:3]: Try to understand what the original `$event.stopPropagation()` is doing. Hint: Click on the `more` button. It is not supposed to expand the tab.

Also, if you open the console, you will find a lot of errors when you expand the tab, due to this line change.

File: src/web/styles.scss [hunk 2]
```diff
@@ -10,6 +10,19 @@ body {
...
+}
+
+tm-view-photo-popover {
+  cursor: default;
+}
+
+.card-header {
+  cursor: pointer;
+}
+
```
Comment (ccyccyccy) [inline src/web/styles.scss:25]: We normally add the styling of components to their specific SCSS files, not to the global file

**Comment by Adrian27045** [comment]
I have made some changes. 
Sorry for my mistakes. I am a beginner in programming and new in this project.
Please review.

**Comment by Adrian27045** [comment]
Sorry, I didn't see it. I have made one change and now it's working.

`This issue is actually more complicated than it seems. That's why we didn't put a d.FirstTimers label on this.`

I know it, but when I looked for an issue there was nothing with d.FirstTimers however, I wanted to solve an issue.

File: src/web/app/pages-instructor/instructor-session-result-page/instructor-session-result-question-view.component.scss [hunk 3]
```diff
@@ -1,3 +1,13 @@
...
+
+// The clickable parts of the boxes should change the cursor to a pointer
+
+.tm-question-text-with-info {
+  cursor: pointer;
+}
+
+.card-header {
+  cursor: pointer;
+}
```
Comment (ccyccyccy) [inline src/web/app/pages-instructor/instructor-session-result-page/instructor-session-result-question-view.component.scss:13]: Not sure why you have 2 of them. 1 should do

Comment (ccyccyccy) [reply src/web/app/pages-instructor/instructor-session-result-page/instructor-session-result-question-view.component.scss:13]: Also, try not to name your classes like `.tm-question-text-with-info`. It is not descriptive

File: src/web/app/pages-instructor/instructor-session-result-page/instructor-session-result-question-view.component.ts [hunk 4]
```diff
@@ -41,4 +41,14 @@ export class InstructorSessionResultQuestionViewComponent
           return val1.question.questionNumber - (val2.question.questionNumber);
         });
   }
+
+  // Method expanding box after click on text or description
+  expandBoxByTextClick(event: any): void {
+
+    if (event.target.tagName === 'A') {
```
Comment (ccyccyccy) [inline src/web/app/pages-instructor/instructor-session-result-page/instructor-session-result-question-view.component.ts:48]: What is this line checking for?

File: src/web/app/pages-instructor/instructor-session-result-page/instructor-session-result-question-view.component.html [hunk 5]
```diff
@@ -1,6 +1,7 @@
 <div class="card margin-bottom-15px" *ngFor="let question of questionsOrder">
   <div class="card-header bg-info text-white course-tab-header" (click)="toggleAndLoadTab.emit(question.question.feedbackQuestionId)">
-    <tm-question-text-with-info [questionNumber]="question.question.questionNumber" [questionDetails]="question.question.questionDetails" (click)="$event.stopPropagation()"></tm-question-text-with-info>
+    <tm-question-text-with-info class="tm-question-text-with-info" [questionNumber]="question.question.questionNumber" [questionDetails]="question.question.questionDetails"
+                                 (click)="expandBoxByTextClick($event)" ></tm-question-text-with-info>
```
Comment (ccyccyccy) [inline src/web/app/pages-instructor/instructor-session-result-page/instructor-session-result-question-view.component.html:4]: On click handler here has overlapped functionality with the on click handler at line 2

**Pull request merged**